Application: Added Brands Functionality

MuhammadAbdullah191 opened the pull request:
-> Added Brand Model
-> Added Brand Controller
-> Added Admin Ability to create, edit and delete Brands

MuhammadAbdullah191 committed:
Application: Added Brands Functionality
-> Added Brand Model
-> Added Brand Controller
-> Added Admin Ability to create, edit and delete Brands

husnainrazadevsinc commented on app/models/brand.rb:
```diff
@@ -0,0 +1,5 @@
+class Brand < ApplicationRecord
+
+	validates :name, presence: true
```
spacing issue

husnainrazadevsinc commented on db/migrate/20230327070447_create_brands.rb:
```diff
@@ -0,0 +1,10 @@
+      t.string :name, null: false
+      t.text :description, null: false
+      
+      t.timestamps
```
if null is false, you should use default value for that attribute as well, please take care of it.

husnainrazadevsinc reviewed (changes requested):
Changes suggested. Please check

MuhammadAbdullah191 committed:
Feedback Addressed:
-> Added Dafault value to Brand model
-> Added blank check to set_brand_function in brand controller
Added Failure check to brand delete action
